packages/app-builder/src/components/Scenario/Sanction/FieldBlackListId.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, MenuCommand, Switch } from 'ui-design-system';
+import { Icon } from 'ui-icons';
+
+interface CustomList {
+  id: string;
+  name: string;
+}
+
+interface FieldBlackListIdProps {
+  value?: string | null;
+  onBlur: () => void;
+  onChange: (value: string | null) => void;
+  editor: 'view' | 'edit';
+  customLists: CustomList[];
+}
+
+export const FieldBlackListId = ({
+  value,
+  onBlur,
+  onChange,
+  editor,
+  customLists,
+}: FieldBlackListIdProps) => {
+  const { t } = useTranslation();
```

Comment:
issue: we miss the desired namespace

```suggestion
  const { t } = useTranslation(['scenarios']);
```